travis.sh

```diff
@@ -30,6 +30,7 @@ install_digger
 dmd --version
 rdmd --help | head -n 1
 dub --version
+ldmd2 --version
```

Comment:
The install script sets `$DC`  (`ldc2`) and `$DMD` (`ldmd2`).
Note that these commands are only necessary because `dmd` is built freshly from source here. I'm about to add this tools repo to the Jenkins Project Tester (https://github.com/dlang/ci/pull/130), so that the test suite will be run on every PR at dmd,druntime and phobos which in other words means that we don't need to build the latest `dmd` here and can just use `$DC` from Travis.

Reply:
FYI here's the PR to dlang/tools: https://github.com/dlang/tools/pull/298